neurokit2/hrv/hrv_nonlinear.py

```diff
@@ -13,7 +13,7 @@
 from .hrv_utils import _hrv_get_rri, _hrv_sanitize_input
 
 
-def hrv_nonlinear(peaks, sampling_rate=1000, show=False, **kwargs):
+def hrv_nonlinear(peaks, sampling_rate=1000, dfa_short_window=[4, 16], dfa_long_window=[16, 64], show=False, **kwargs):
```

Comment:
not sure if there is a better way of naming these arguments 😄

Reply:
`dfa_windows = [[4, 16], [16, 64]]`? Not the best to have nested lists but... It's extendable and parsimonious in terms of arguments number